mmv1/products/cloudrunv2/Job.yaml

```diff
@@ -226,6 +233,12 @@ properties:
         name: 'useDefault'
         description: |
           If True, indicates to use the default project's binary authorization policy. If False, binary authorization will be disabled.
+  - !ruby/object:Api::Type::String
+    name: start_execution_token
```

Comment:
Do we want to include `run_execution_token` in the same PR? That might be the more common user scenario (they are oneof)

Reply:
okay it look like that one isn't visible for v2 yet, we'll do that in the future.